homeassistant/components/ipma/weather.py

```diff
@@ -125,27 +161,32 @@ def condition(self):
     @property
     def temperature(self):
         """Return the current temperature."""
-        return self._condition.temperature
+        if self._condition:
+            return self._condition.temperature
 
```

Comment:
Always return something explicitly if we `return` something anywhere. `return None` by default.